Sources/ForageSDK/Foundation/ForageSDK+ForageServices.swift

```diff
@@ -145,6 +145,8 @@ extension ForageSDK: ForageSDKService {
                 ))
                 .notice("Called ForageSDK.shared.capturePayment for Payment \(paymentReference)", attributes: nil)
             
+            foragePinTextEdit.disable()
```

Comment:
Why did you move these `disable` calls below the log calls?

Reply:
We want the span/trace to start as soon as the method is called.

e.g. timestamp of "Called ForageSDK.shared.capturePayment" matches the same moment "capturePayment" was called.

Suppose "disable" was called first and it took a while to complete executing or threw some kind of error, we would have inaccurate timestamps and/or the "capturePayment" would never get logged.